README.md

```diff
@@ -73,6 +78,9 @@ url = "https://github.com/username/package/pull/139"
 
 # Mandatory: name of the affected package on Hackage
 package = "acme-broken"
+# OR: name of the affected GHC component; one of:
+# ghc | ghci | rts | ghc-pkg | runghc | ghc-iserv | hp2ps | hpc | hsc2hs | haddock
+ghc-component = "ghci"
```

Comment:
The README is also validated by the CI, so this key should be commented.

Reply:
Thank you, I'll fix it now.